src/org/opensolaris/opengrok/history/FileHistoryCache.java

```diff
@@ -309,12 +303,45 @@ private void storeFile(History histNew, File file, Repository repo) throws Histo
                 }
             }
         } catch (IOException ex) {
-            // Ideally we would want to catch the case when incremental update
-            // is done but the cached file is not there however we do not have
-            // the data to do it here.
```

Comment:
I think this sort of info belongs to Issues, not to code. Moreover, the feasibility of this change is low so such comment does not have much value.